WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -395,12 +395,13 @@ private void initAnalytics(final long elapsedTimeOnCreate) {
     }
 
     /**
-     * Application.onCreate is called before any activity, service, or receiver - it can be called while the app
-     * is in background by a sticky service or a receiver, so we don't want Application.onCreate to make network request
+     * Application.onCreate is called before any activity, service, or receiver.
```

Comment:
This whole comment block change also changed the message the original was trying to convey. Can we change it back? 🙂